OpenSim/Simulation/Model/Frame.h

```diff
@@ -185,6 +193,11 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Frame, ModelComponent);
     */
     SimTK::Transform findTransformInBaseFrame() const;
 
+    /** Accessor for position of Frame in Ground frame as output. */
```

Comment:
Agreed @chrisdembia on the need to unify names :+1:
I left the reference to Output due to the discussion regarding what's allowed as output function, but I'm happy to remove it now.
If Output functions will change semantics to mean more than a plain wire/bind then we should give them a distinct name and include comment to signify that.